Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -151,6 +141,15 @@ private void OnUnloaded(object sender, RoutedEventArgs e)
         }
 
         /// <summary>
+        /// If a selection is set, this updates the view to show it. By default, this is called anytime
+        /// a selection changes.
+        /// </summary>
+        protected virtual void ShowSelection()
```

Comment:
Not sure this really offers any value. Consider your usecase of subscribing to the ItemClick event. This control still needs to work in an expanded view. I would expect that if I arrow up/down when expanded that the detail would change. This is only a problem when in a collapsed state. I believe this is the only use case where we need to worry about keyboard navigation not changing selection